eox_core/cms/views.py

```diff
@@ -146,9 +238,21 @@ def get_json_from_body_request(request):
     Function to convert request.body into a json format.
     """
     json_content = {}
-    if "application/json" in request.META.get('CONTENT_TYPE', '') and request.body:
+    if 'application/json' in request.META.get('CONTENT_TYPE', '') and request.body:
         try:
             json_content = json.loads(request.body)
         except ValueError:
             return json_content
     return json_content
+
+
+def get_course_regex(input_course_regex):
```

Comment:
I'm not sure I understand the reason for writing this function. Is it because the original regex is not url safe and you need to do some replacements? what about doing a url_encode first then?

If the reason is different, then please clarify in the docstring.